src/libs/error.js

```diff
@@ -1,30 +1,12 @@
-import _ from 'lodash/fp'
+import { notify, sessionTimeoutProps } from 'src/components/Notifications'
 import { reloadAuthToken, signOut } from 'src/libs/auth'
-import * as StateHistory from 'src/libs/state-history'
-import * as Utils from 'src/libs/utils'
 
 
-export const errorStore = Utils.atom([])
-
-const addError = item => errorStore.update(state => _.concat(state, [item]))
-
 export const reportError = async (title, obj) => {
   if (obj instanceof Response && obj.status === 401 && !(await reloadAuthToken())) {
-    addError({ title: 'Session timed out', error: 'You have been signed out due to inactivity', code: 'sessionTimeout' })
+    notify('Session timed out', sessionTimeoutProps)
```

Comment:
Needs a `type` argument.

Reply:
Another good catch! That was related to a relatively late refactor.